notebooks/nightly_report/nightly_report_aos_eas.ipynb

```diff
@@ -0,0 +1,1027 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #105.        ring5_data = df[f"mean_5"]
</pre><p>you don't need f here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>